src/schema/v2/conversation/message.ts

```diff
@@ -134,5 +134,13 @@ export const MessageType = new GraphQLObjectType<any, ResolverContext>({
       resolve: ({ metadata }) => isInvoiceMessage(metadata),
     },
     createdAt: date(),
+    to: {
+      description: "Masked emails w/ display name of the recipients.",
+      type: new GraphQLList(GraphQLString),
+    },
+    cc: {
+      description: "Masked emails w/ display name of the recipients in copy.",
+      type: new GraphQLList(GraphQLString),
```

Comment:
These types could and should be changed to `new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(GraphQLString))))`

Reply:
I thought [those two can be `null`](https://github.com/artsy/radiation/blob/main/db/schema.rb#L133-L134), no?

Reply:
Should they just be empty array if they are `null`?

Reply:
yeah, I think `new GraphQLList(new GraphQLNonNull(GraphQLString)))` would suit them better, right?

Reply:
Generally; if you're expecting empty array then marking both the array and the array contents as non-nullable will make this type accurate and much easier to work with. If you're actually expecting null; then null; though in this case it looks like empty array would be preferred.

Reply:
> "if you're expecting empty array then marking both the array and the array contents as non-nullable will make this type accurate and much easier to work with"

I would've thought that marking both as non-nullable would not allow for an empty array. I might be reading this sentence wrong, though.

> "If you're actually expecting null; then null; though in this case it looks like empty array would be preferred."

I think an empty array would be preferred, yep.

Reply:
Consider the difference between: `[]` and `[null, 'whatever']`

There's nothing about empty array that means you need to mark anything as nullable. `const foo: string[] = []` is totally valid right?

Reply:
Makes total sense. Thanks for putting it into perspective 👍  Will make the change tomorrow morning.